cli.py

```diff
@@ -0,0 +1,23 @@
+import click
+from asaps import models
+from asnake.client import ASnakeClient
+
+
+@click.command()
+@click.option('-u', '--url', envvar='ARCHIVESSPACE_URL')
```

Comment:
This is optional, but it's pretty common for scripts that fetch things to use `-u/--user` and `--url` (no short option). curl is a good example.

Reply:
That's what I originally had, I'll revert back to that